server/test/api/functions.js

```diff
@@ -0,0 +1,94 @@
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+var uuid    = require('node-uuid');
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+/**
+* The /function  API endpoint
+*
+* This test suite implements full CRUD on the /function  HTTP API endpoint.
+*/
+describe('The /function  API endpoint', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+      // Function we will add during this test suite.
+  var fonction = {
+    fonction_txt : 'Anestiologiste'
+  };
+ 
+  var FUNCTION_KEY = ['id', 'fonction_txt'];
+
+  var DEFAULT = 2;
+
+  // login before each request
+  beforeEach(helpers.login(agent));
+
+
+  it('GET /function  returns a list of function ', function () {
```

Comment:
You are actually sending a GET request to `/functions` not `/function`.  Please make sure your test descriptions are accurate, we will need them when we debug!